format/index.md

```diff
@@ -513,6 +513,39 @@ the following members:
   data represents a resource relationship.
 * Pagination links for the primary data (as described below).
 
+Value of each member of a `links` object can be either a string
+containing the link URL or a "link object", which can contain
+the following members:
+
+* `"href"` - the string containing the link URL or an array containing multiple strings containing the link URLs,
```

Comment:
> Since we are only reserving self and related currently, and neither of those require an array, we don't need to speak normatively about this, and the window is still open for supporting arrays on new key later. Sound good @ethanresnick / @hhware ?

Yeah, that works for me for 1.0! Thanks @tkellen, and sorry for my scatterbrainedness on this :)